lego-mindstorms-pieces.py

```diff
@@ -190,26 +221,42 @@ def order(shop=None, browser=None, lego_set=None, order_list=None, username=None
     age_field.send_keys('55')
     age_field.send_keys(Keys.RETURN)
 
+    # wait for age_field's DOM element to be removed
+    wait.until(EC.staleness_of(age_field))
+
+    ## login stuff
     if username and password:
+
         print("Let's log in with LEGO ID {user}.".format(user=username))
-        login_link = browser.find_element_by_css_selector('.legoid .links > a')
+        login_link = wait.until(EC.element_to_be_clickable((By.CSS_SELECTOR,".legoid .links > a[data-uitest = 'login-link']")))
```

Comment:
- Please wrap the line before 80 characters line length.
- Please use a space after the comma.
- Please don't use spaces around the `=` sign in the CSS selector.